package.json

```diff
@@ -180,7 +180,7 @@
     "react-dom": "^18.2.0",
     "react-error-boundary": "^3.1.4",
     "react-helmet-async": "^1.3.0",
-    "react-hook-form": "^7.33.1",
+    "react-hook-form": "^7.51.5",
```

Comment:
A bunch of async validation fixes in 7.39.0 so figured out this would be the right time to bump to latest while at it